homeassistant/components/overkiz/entity.py

```diff
@@ -93,14 +94,28 @@ class OverkizSensorDescription(SensorEntityDescription):
     ] | None = lambda val: val
 
 
+@dataclass
+class OverkizNumberDescriptionMixin:
+    """Define an entity description mixin for number entities."""
+
+    command: str
+
+
+@dataclass
+class OverkizNumberDescription(NumberEntityDescription, OverkizNumberDescriptionMixin):
+    """Class to describe an Overkiz number."""
+
+
```

Comment:
I think it would be better to store these inside the number module.

Reply:
Addressed in d8d7d3f